src/partners/simplex.ts

```diff
@@ -140,16 +142,34 @@ export const simplex: PartnerPlugin = {
 
 export function processSimplexTx(rawTx: unknown): StandardTx {
   const tx = asSimplexTx(rawTx)
+
+  const depositCurrency = tx.currency
+  const payoutCurrency = tx.crypto_currency
+  const isDepositFiat = isFiatCurrency(depositCurrency)
+  const isPayoutFiat = isFiatCurrency(payoutCurrency)
+
+  const direction = isDepositFiat ? 'buy' : isPayoutFiat ? 'sell' : undefined
+
+  if (direction == null) {
+    throw new Error(
+      `Unknown direction for tx ${tx.order_id}; no fiat currency in trade from ${depositCurrency} to ${payoutCurrency}`
+    )
+  }
+
   const standardTx: StandardTx = {
     status: 'complete',
     orderId: tx.order_id,
+    countryCode: tx.country,
     depositTxid: undefined,
     depositAddress: undefined,
-    depositCurrency: tx.currency,
+    depositCurrency,
     depositAmount: safeParseFloat(tx.fiat_total_amount),
+    direction,
+    exchangeType: 'fiat',
+    paymentType: null,
```

Comment:
really, no paymenttype? We need to move to the new simplex API and we'll probably get that info

Reply:
Yes, I don't see any payment type/method information in the rawTx